app/core/initialize.coffee

```diff
@@ -3,6 +3,18 @@ app = null
 utils = require './utils'
 { installVueI18n } = require 'locale/locale'
 
+VueRouter = require 'vue-router'
+Vuex = require 'vuex'
```

Comment:
We already have vuex being initialized in `vendor.js`. So this might not be required.

Reply:
Should we move these all to vendor JS then?  What is the purpose of Vendor JS / do they belong there?

Reply:
`vendor.js` sets some global variables on `window` object, in addition to importing some scripts as well.

Reply:
Here's the offending line in [vendor.js](https://github.com/codecombat/codecombat/blob/43a690b8f1f35b54128c2f1ccee9991bb13188fa/app/vendor.js#L35).

I wonder how it interacts with what is defined here.

Reply:
Do we have a `Vue.use(Vuex)` call anywhere?

Reply:
Nope, I could not find it in the codebase anywhere.